maintenance/shaka-player.md

```diff
@@ -0,0 +1,185 @@
+   break semantic versioning.
+
+
+## Release Policy
+
+Maintainers can make a feature release at almost any time.  There is no need to
+wait to batch up any particular number of changes or any particular significant
+features.
+
+
+## Branch Maintenance Policy
+
+We will cherry-pick bug fixes to LTS branches, plus the most recent 2 release
+branches.
+
+There are two kinds of branches we will dub LTS:
+ - The last feature release (minor release number) before a breaking release
+   (major release number) will receive fixes for 1 year
+   - _For example, v3.3 (right before v4.0) will be fixed for 1 year past the
+     release of v4.0.0 (until April 30, 2023)._
+ - The release currently used by the Cast Application Framework will continue
+   to receive fixes until CAF upgrades to a newer Shaka Player release by
+   default
+   - _As of June, 2022, this is currently v3.2._
+
+In addition to LTS branches, the most recent two branches will always receive
+fixes.  If the most recent release branch is v4.1, the most recent two branches
+would be v4.1 and v4.0.  After v4.2 is released, the most recent two branches
+would become v4.2 and v4.1.  Unless v4.0 was the current CAF branch, we would
+stop fixing v4.0 when v4.2 is released.
+
+
+## When to Make Breaking Changes
+
+We should make breaking changes infrequently if we can.  The goal should be one
+breaking release per year at most.  If changes can be made in a
+backward-compatible way (with a shim or behind a non-default configuration),
+they should be.  Then they can be released in a normal feature release.
+
+Deprecated features, shims, and temporary configurations can be removed when a
+major (breaking) release is finally made.
+
+For example, in v4.0, we were forced to make a change that disabled HLS support
+for older smart TVs.  In that same release, we removed deprecated methods,
+configurations that were added to maintain compatibility with older releases,
+etc.  Many of them had been marked deprecated for 6-12 months already.
+
+Breaking changes need to be listed in
+[`docs/tutorials/upgrade.md`](https://github.com/shaka-project/shaka-player/blob/main/docs/tutorials/upgrade.md),
+with details on what application developers will need to do to cope with each
+change.
+
+
+## Internal Release Processes
+
+After making public releases, the following internal steps are taken at Google
+to make the release available in our internal systems.  Detailed documents on
+these processes are internal-only (go/shaka-player).
+
+ 1. Update the Chromecast receiver app URLs for the Shaka Player Demo
+
+    > :pencil: **TODO**: Automate this!
+
+ 1. Update Google3 (internal source repo)
+
+    > :pencil: **TODO**: Automate this?
+
+ 1. Update [Google Hosted Libraries](https://developers.google.com/speed/libraries)
+
+    > :pencil: **TODO**: Automate this?
+
+ 1. Announce the release via email
+
+    > :pencil: **TODO**: Automate this or deprecate the practice!
```

Comment:
I am in favor of eliminating this practice, people can already subscribe to the GitHub to obtain updates

Reply:
Ack.  I already failed to email on the last couple of releases.  I think what I need to do now is email the list and announce the end of email announcements.  :-)  I'll send out instructions for subscribing via GitHub.

Reply:
Thanks!

Reply:
Merged in player, and packager, and announced via email.